core/spec/models/spree/preferences/static_model_preferences_spec.rb

```diff
@@ -13,6 +13,8 @@ module Spree
     let(:other_class){ Class.new }
     let(:definitions){ subject.for_class(preference_class) }
 
+    before { stub_const("SomeClass", preference_class) }
```

Comment:
We only need this stub within the new test case. What do you think about eliminating it and interpolating `preference_class.to_s` in the error message?

Reply:
Done, thanks for the suggestion!